pageserver/src/tenant/timeline.rs

```diff
@@ -117,65 +118,11 @@ impl PartialOrd for Hole {
     }
 }
 
-pub struct LayerMapping(HashMap<PersistentLayerKey, Arc<dyn PersistentLayer>>);
+pub struct LayerMapping(());
```

Comment:
Why is this left behind? It would appear that all lines which would be touched by removal of the tuple would be touched by just getting rid of this.

Reply:
This can help reduce diffs to the upstream PR so that I will spend less time resolving conflicts 🤣 I prefer to remove this in next PR, or when all reviews are resolved for this PR, I can append a final commit to remove this.